lxd/auth_groups.go

```diff
@@ -181,6 +181,15 @@ func getAuthGroups(d *Daemon, r *http.Request) response.Response {
 		return response.SmartError(fmt.Errorf("Failed to get a permission checker: %w", err))
 	}
 
+	withEntitlements, err := extractEntitlementsFromQuery(r, entity.TypeAuthGroup)
+	if err != nil {
+		return response.SmartError(err)
+	}
+
+	if len(withEntitlements) > 0 && recursion != "1" {
```

Comment:
```suggestion
	if len(withEntitlements) > 0 && !recursion {
```
Should be more future-proof this way in case higher recursion levels are introduced. wdyt?

Reply:
fixed